xtask/src/main.rs

```diff
@@ -798,9 +808,11 @@ fn run_cargo_udeps(scope: &Scope) -> Step {
 
     Step::Multiple {
         name: "cargo udeps".to_string(),
-        steps: workspace_manifest_files()
+        steps: crate_manifest_files()
```

Comment:
Just checking: I seem to vaguely remember that switching from compiling crates to compiling workspaces was faster; could you check there are no major regressions by undoing that?

Reply:
Fair point, updating this to keep using workspaces via the `--exclude` flag then